server/src/test/java/io/spine/server/entity/DefaultStateRegistryShould.java

```diff
@@ -24,22 +24,94 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import static com.google.common.collect.Maps.newConcurrentMap;
+import static io.spine.Identifier.newUuid;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
 /**
  * @author Alexander Yevsyukov
+ * @author Dmitry Ganzha
  */
 public class DefaultStateRegistryShould {
 
+    private static final String DEFAULT_STATES_FIELD_NAME = "defaultStates";
     private DefaultStateRegistry registry;
+    private Map<Object, Object> spyMap;
 
     @Before
     public void setUp() {
-        registry = new DefaultStateRegistry();
+        spyMap = spy(newConcurrentMap());
+        registry = DefaultStateRegistry.getInstance();
+        try {
```

Comment:
Please extract this `try {} catch {}` into a method. E.g. `injectField(Object target, String fieldName, Object injectableValue)`. 

In this way the code will be more self-documenting than now.